packages/core/test/lib/tracing/trace.test.ts

```diff
@@ -399,6 +399,40 @@ describe('startSpan', () => {
     });
   });
 
+  it('allows to pass span links', () => {
```

Comment:
sorry for the nit but I think this describes it a little better, given that this test is in the `describe('startSpan')` closure. 

```suggestion
  it('allows to add span links', () => {
```